CONTRIBUTING.md

```diff
@@ -49,7 +49,8 @@ We use GitHub issues to track public bugs. Report a bug by [opening a new issue]
 People *love* thorough bug reports. I'm not even kidding.
 
 ## Coding Style
-Please write clear and well doccumented code.
+- Run cargo fmt in the project directory to automatically format code
```

Comment:
very small comment: you can use backticks to format text as code. so maybe `cargo fmt` and `cargo clippy` here.

Reply:
but I'll merge as is, we can take another pass later as this is not crucial. thanks!